src/configs/firebaseConfigs.js

```diff
@@ -0,0 +1,11 @@
+const firebaseConfigs = {
```

Comment:
In the long run you need to move these values into environment variables so you can differentiate testing and production apps, as referenced in the docs: https://firebase.google.com/docs/projects/api-keys#test-vs-prod-keys